src/main/java/org/terasology/segmentedpaths/events/OnExitSegment.java

```diff
@@ -18,14 +18,27 @@
 import org.terasology.entitySystem.entity.EntityRef;
 import org.terasology.entitySystem.event.Event;
 
+/**
+ * Event called when an entity which has {@link org.terasology.segmentedpaths.components.PathFollowerComponent} leaves a segment.
+ */
 public class OnExitSegment implements Event {
 
+    /**
+     * EntityRef to an entity, which leaves a segment.
+     */
     private EntityRef pathFollowingEntity;
 
+    /**
+     * Event constructor.
+     * @param pathFollowingEntity which leaves a segment.
```

Comment:
Again, you can't use param name as part of the javadoc.

Reply:
Also, I feel like `* @param pathFollowingEntity The entity that left the segment.` makes more sense, since this event gets generated AFTER the entity has left the segment. I've discussed this with @pollend and he agrees.